cmd/zed/auth/login.go

```diff
@@ -51,42 +58,55 @@ func (c *LoginCommand) Run(args []string) error {
 	switch method.Kind {
 	case api.AuthMethodAuth0:
 	case api.AuthMethodNone:
-		return fmt.Errorf("Zed lake service at %s does not support authentication", c.Lake)
+		return fmt.Errorf("Zed service at %s does not support authentication", c.Lake)
 	default:
-		return fmt.Errorf("Zed lake service at %s requires unknown authentication method %s", c.Lake, method.Kind)
+		return fmt.Errorf("Zed service at %s requires unknown authentication method %s", c.Lake, method.Kind)
 	}
 	fmt.Println("method", method.Auth0.ClientID)
 	fmt.Println("domain", method.Auth0.Domain)
 	fmt.Println("audience", method.Auth0.Audience)
-	dar, err := devauth.DeviceAuthorizationFlow(ctx, devauth.Config{
-		Audience: method.Auth0.Audience,
-		Domain:   method.Auth0.Domain,
-		ClientID: method.Auth0.ClientID,
-		Scope:    "openid profile email offline_access",
-		UserPrompt: func(res devauth.UserCodePrompt) error {
-			fmt.Println("Complete authentication at:", res.VerificationURL)
-			fmt.Println("User verification code:", res.UserCode)
-			if c.launchBrowser {
-				browser.OpenURL(res.VerificationURL)
-			}
-			return nil
-		},
-	})
+	auth0client, err := auth0.NewClient(*method.Auth0)
+	if err != nil {
+		return err
+	}
+	dcr, err := auth0client.GetDeviceCode(ctx, deviceCodeScope)
```

Comment:
Nit: Ditch deviceCodeScope.
```suggestion
	dcr, err := auth0client.GetDeviceCode(ctx, "email offline_access openid profile")
```